lib/rubocop/cop/cop.rb

```diff
@@ -132,6 +132,12 @@ def display_cop_names?
           config['AllCops'] && config['AllCops']['DisplayCopNames']
       end
 
+      def display_style_guide?
+        style_guide_url &&
+          (debug? || @options[:display_style_guide] ||
```

Comment:
Got it - the name is displayed there because when debugging people often have to find out what's the problematic cop. Showing the url won't help much IMO.